site/en/tutorials/embeddings_quickstart.ipynb

```diff
@@ -37,7 +37,7 @@
         "id": "title"
```

Comment:
<p>This file was deleted at HEAD</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>